nimbus/src/enrollment.rs

```diff
@@ -51,8 +326,13 @@ pub enum EnrollmentStatus {
 }
 
 impl EnrollmentStatus {
-    // This is used in examples, but not in the main dylib.
-    #[allow(dead_code)]
+    pub fn enrolled(reason: EnrolledReason, branch: &str) -> Self {
```

Comment:
nit: could something like `new_enrolled` make it slightly more clear that this is a constructor?